src/webgl/framebuffer.js

```diff
@@ -77,37 +58,37 @@ export default class Framebuffer {
 
     log.log(2, `Resizing framebuffer ${this.id} to ${width}x${height}`);
 
-    const {gl} = this;
+    this.opts.colorBuffer = null;
+    this.opts.depthBuffer = null;
+    this.opts.stencilBuffer = null;
+    this.opts.texture = null;
 
     // TODO - do we need to reallocate the framebuffer?
-    const colorBuffer = new Texture2D(gl, {
-      minFilter: this.minFilter,
-      magFilter: this.magFilter
-    })
-    // TODO - should be handled by Texture2D constructor?
-    .setImageData({
+    const colorBuffer = new Texture2D(this.gl, {
       data: null,
+      format,
+      type,
       width,
       height,
-      type,
-      format
+      [GL.TEXTURE_MIN_FILTER]: this.minFilter,
```

Comment:
The dynamic object keys? They allow us to use a variable (in this case a "constant") as the key in an object.

```
const a = "dynamicKey";
const obj = {[a]: "value"}
```

```
var a = "dynamicKey";
var obj = {};
obj[a] = "value";
```

The idea is that instead of defining hundreds of new luma.gl specific keys (in this case e.g. 'textureMinFilter`) we just use the GL constants people are already familiar with.